media/livestream/pom.xml

```diff
@@ -36,7 +36,8 @@
     <maven.compiler.source>11</maven.compiler.source>
   </properties>
   <!--  Using libraries-bom to manage versions.
-  See https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM -->
+  See
+  https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM -->
```

Comment:
(nit) Move "See" to the previous line

<!-- Using libraries-bom to manage versions. See
https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM -->